src/main/java/mate/academy/hwspringbootintro/model/CartItem.java

```diff
@@ -0,0 +1,41 @@
+package mate.academy.hwspringbootintro.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.Table;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Positive;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Entity
+@Getter
+@Setter
+@Table(name = "cart_items")
+@NoArgsConstructor
+public class CartItem {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "shopping_cart_id", nullable = false)
+    private ShoppingCart shoppingCart;
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "book_id", nullable = false)
+    private Book book;
+    private int quantity;
+
+    public CartItem(@NotNull ShoppingCart shoppingCart,
+                    @NotNull Book book,
+                    @NotNull @Positive int quantity) {
```

Comment:
why do you need this constructor? 
```suggestion
    public CartItem(ShoppingCart shoppingCart,
                    Book book,
                    int quantity) {
```